_infra/helm/sample-file-uploader/Chart.yaml

```diff
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 1.0.0
+version: 0.0.25
```

Comment:
Hmm I don't think we so go backwards. In this one I'd be tempted to move the app version forward to 1.0.0